keps/sig-node/3675-resource-plugin-manager/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: Pluggable Resource Management
+kep-number: 3675
+authors:
+  - "@obiTrinobiIntel"
+  - "@catblade"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
```

Comment:
for the next upload: should be `implementable`

Reply:
done